src/AccessibilityInsights.SharedUx/FileIssue/IssueReporterManager.cs

```diff
@@ -35,19 +34,21 @@ public static IssueReporterManager GetInstance()
             return _defaultInstance;
         }
 
+        // Production constructor
         private IssueReporterManager()
+            : this(ConfigurationManager.GetDefaultInstance().AppConfig, Container.GetDefaultInstance().IssueReportingOptions?.ToList())
+        {
+        }
+
+        // Unit testing constructor
+        internal IssueReporterManager(ConfigurationModel configs, List<IIssueReporting> issueReportingOptions)
```

Comment:
>List [](start = 66, length = 4)

IEnumerable instead of List? That would remove the need to call ToList in the production constructor